openapi.yaml

```diff
@@ -336,6 +336,8 @@ paths:
                           type: string
                           description: Single date to return appointments for
                           format: date
+                loadingType:
```

Comment:
We discussed not placing loading type as a filter option currently so this should be removed here.